src/router.tsx

```diff
@@ -128,6 +128,7 @@ function AppRouter() {
           <Route path="/pgraph/:agent" element={<ForceGraph />} />
           <Route path="/ipfs" element={<IpfsSettings />} />
           <Route path="/ipfs/:cid" element={<Ipfs />} />
+          <Route path="/ipfs/:cid/:tab" element={<Ipfs />} />
```

Comment:
lets decide about `const` routes

Reply:
or inner routes for such?

Reply:
For me at this moment is much understandable to see full view of routing at 1 place

Reply:
need to talk with @dimakorzhovnik and decide, later